Elevel/Domain/Models/Question.cs

```diff
@@ -9,17 +9,13 @@ namespace Elevel.Domain.Models
 {
     public class Question : BaseDataModel
     {
-        public string NameQuestion { get; set; }
+        public string QuestionName { get; set; }
```

Comment:
you can simplify just Name, because you will already have question as object.
question.Name looks better than
question.QuestionName
and I would like to see Text/Value, because question itself has no name, but I'm OK with your decision.